app/models/authorized_user.rb

```diff
@@ -1,26 +1,19 @@
 # == Schema Information
 #
-# Table name: Authorized_users
+# Table name: authorized_users
 #
 #  id                     :integer          not null, primary key
 #  email                  :string           default(""), not null
-#  role                   :string           default("coach"), not null
 #
 
-class Authorized_user < ActiveRecord::Base
-
-  ADMIN = "admin"
-  COACH = "coach"
-
+class AuthorizedUser < ActiveRecord::Base
+  validates_format_of :email,:with => Devise::email_regexp
+  
   def self.has_email?(email)
-    if self.find_by(email: email).nil?
+    if self.find_by_email(email).nil?
```

Comment:
hmm does this `has_email?` function work? shouldn't it be `AuthorizedUser.find_by_email(...)`